functional/Vagrantfile

```diff
@@ -0,0 +1,145 @@
+# -*- mode: ruby -*-
```

Comment:
The only one string we added is `config.vm.provision :shell, :path => "provision/setup.sh", :privileged => false`. The rest is original coreos-vagrant Vagrantfile.